cypress/integration/test-helper.js

```diff
@@ -0,0 +1,13 @@
+export const testRequestResponse = (uri, responseCode) => {
```

Comment:
This PR should only add e2es for Storybook and we should refactor other code in another PR. Because of this, while this function has value, it should be created in that PR. Should just test the response status in the storybook spec manually.

Reply:
The reason for this function abstraction is Code climate was complaining about duplicate code between article_spec and storybook_spec. So going forward we either manually check storybook and lower our maintainability value in code climate or we handle it within this PR. Happy with either approach just something we should discuss with the team IMO

Reply:
Okay. If the refactor was anymore substantial and not causing CC issues I'd say to refactor in another PR.